core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -2123,29 +2123,35 @@ public int getMaxSIRepairLimit(String dbName, String tableName) {
    * folder will take place
    */
   private void validateTrashFolderRetentionTime() {
-    String propertyValue = carbonProperties.getProperty(CarbonCommonConstants
-        .CARBON_TRASH_RETENTION_DAYS, Integer.toString(CarbonCommonConstants
-        .CARBON_TRASH_RETENTION_DAYS_DEFAULT));
+    String propertyValue = carbonProperties.getProperty(
```

Comment:
can you use `getTrashFolderRetentionTime` here also? so you avoid integer parsing again

Reply:
@akashrn5 : getTrashFolderRetentionTime is used to access the already validated and stored value. It won't check the parsing errors as it is already validated. so current changes looks ok for me for this first time validation for add property

Reply:
@ajantha-bhat the `getTrashFolderRetentionTime ` method implementation is just IntegerparseInt(Carbonproperties.getproperty) so, here also its the same thing right. What exactly you mean its already stored and not validation?

Reply:
done